docs/app/pages/components/components-sections/icon/icon/icon.component.html

```diff
@@ -0,0 +1,139 @@
+<div class="ux-icon-component-example">
```

Comment:
I think we can simplify this a little bit: https://plnkr.co/edit/ZJcHLEYlgp42n4lN9exY?p=preview
Its also good to avoid the `ux` prefix in sample code classes as we sometimes use this to essentially namespace our styles to avoid class collisions.

Reply:
Updated to plunker example